CMakeLists.txt

```diff
@@ -33,7 +35,7 @@ endif()
 include(DefinePlatform)
 
 # Project definition
-project(az LANGUAGES CXX)
+project("azure-sdk" LANGUAGES CXX)
```

Comment:
We do, but we can remove it. This is not the goal of this PR.
I think having root file is still convenient, because otherwise you need other azure packages being installed and you consume them from the vcpkg, which could be an inconvenience if you need features that were not yet released, so then you would need to fork vcpkg and do what I did to test this, and this is too much time consuming.